admin/content-popup-template.html.php

```diff
@@ -23,21 +24,30 @@
 <template data-opinionstage-content-popup-template>
 	<div class='opinionstage-content-popup-contents' data-opinionstage-content-popup data-opinionstage-client-logged-in="<?php echo esc_attr( $opinionstage_user_logged_in ); ?>">
 		<header class='header'>
-			<div class='header__container'>
+			<div class='header__container <?php echo $is_my_items_admin_page ? 'mw-1000' : ''; ?>'>
 				<div class='header__logo'>
 					<a href='<?php echo esc_url( OPINIONSTAGE_SERVER_BASE ); ?>' target='_blank'>
 						<img src='<?php echo esc_url( plugins_url( 'admin/images/os-logo-header.png', plugin_dir_path( __FILE__ ) ) ); ?>'>
 					</a>
 				</div>
 				<div class='header__action'>
-					<div class='btn-close' @click="closePopup">&#x2715;</div>
+					<?php if ( $is_my_items_admin_page ) { ?>
```

Comment:
conditionals in shared implementation is a strong sign of incorrect architectural direction, let's discuss this in details